client/src/main.jsx

```diff
@@ -14,6 +15,7 @@ ReactDOM.createRoot(document.getElementById('root')).render(
                 <SelectedBoardProvider>
                     <ModalProvider>
                         <App />
+                        <Footer />
```

Comment:
Remove the Footer from Main.jsx and move it back into App. 

`Main.jsx` should be reserved for Providers.

Reply:
Unfortunately, I was unable to verify the other components as I lacked access to them. The issue report indicates that the feature must be visible in all components, but it appears to be already present.

Reply:
I'm not understanding. 

I think the goal here for the issue is just so that the Header and Footer are both aligned/shown on each 'page' that we have. So if the Header shows up in the dashboard, then the footer should also be in the dashboard. 

Whatever way you do it - I don't really care but I think that's the goal here for this issue.

Reply:
If I put Footer in the Dashboard, then I will have to put in the LandingPage too. I can see there is a Header in the Dashboard and LandingPage. Isn't it redundant?